Formula/lua.rb

```diff
@@ -12,93 +11,65 @@ class Lua < Formula
     sha256 "29553ce97e43d24c7b35eb9a44800e0e2b94e83ac79dff25d6d0d0b58590ad7e" => :el_capitan
   end
 
-  pour_bottle? do
-    reason "The bottle needs to be installed into /usr/local."
-    # DomT4: I'm pretty sure this can be fixed, so don't leave this in place forever.
-    # https://github.com/Homebrew/legacy-homebrew/issues/44619
-    satisfy { HOMEBREW_PREFIX.to_s == "/usr/local" }
-  end
-
-  option "with-completion", "Enables advanced readline support"
-  option "without-sigaction", "Revert to ANSI signal instead of improved POSIX sigaction"
   option "without-luarocks", "Don't build with Luarocks support embedded"
 
   # Be sure to build a dylib, or else runtime modules will pull in another static copy of liblua = crashy
   # See: https://github.com/Homebrew/legacy-homebrew/pull/5043
+  # ***Update me with each version bump!***
   patch :DATA
 
-  # completion provided by advanced readline power patch
-  # See http://lua-users.org/wiki/LuaPowerPatches
-  if build.with? "completion"
-    patch do
-      url "https://luajit.org/patches/lua-5.2.0-advanced_readline.patch"
-      sha256 "33d32d11fce4f85b88ce8f9bd54e6a6cbea376dfee3dbf8cdda3640e056bc29d"
-    end
-  end
-
-  # sigaction provided by posix signalling power patch
-  if build.with? "sigaction"
-    patch do
-      # original patch file (not available): http://lua-users.org/files/wiki_insecure/power_patches/5.2/lua-5.2.3-sig_catch.patch
-      url "https://raw.githubusercontent.com/Homebrew/patches/d674e02d1097b21032198854080204680b616b61/lua/lua-5.2.3-sig_catch.patch"
-      sha256 "f2e77f73791c08169573658caa3c97ba8b574c870a0a165972ddfbddb948c164"
-    end
-  end
-
-  # Don't use the https://luarocks.org/releases/luarocks-x.y.z.tar.gz URL
-  # directly as it redirects to the HTTP version of the below URL.
   resource "luarocks" do
-    url "https://luarocks.github.io/luarocks/releases/luarocks-2.4.3.tar.gz"
-    sha256 "4d414d32fed5bb121c72d3ff1280b7f2dc9027a9bc012e41dfbffd5b519b362e"
+    url "https://keplerproject.github.io/luarocks/releases/luarocks-2.3.0.tar.gz"
```

Comment:
This wouldn't need downgrading if you keep the executable of the `lua` formula as `lua` and version the others.